src/SwedbankPay.Sdk/PaymentInstruments/MobilePay/IMobilePayPayment.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+
+namespace SwedbankPay.Sdk.PaymentInstruments.MobilePay
+{
+    /// <summary>
+    /// Object describing a payment for the Mobile Pay payment instrument.
+    /// </summary>
+    public interface IMobilePayPayment

```

Comment:
This probably needs to be done across the board.

```suggestion
    public interface IMobilePayPayment : IPaymentInstrument
```